### ISSUE TITLE
allow key overrides

### ISSUE THREAD

**Issue by sstarcher** [roboll/helmfile#657]

**Comment by mumoshu** [comment]
@sstarcher Good catch! Seems like we need to avoid the go-yaml strict mode or upgrade to go-yaml v3 https://github.com/go-yaml/yaml/issues/410#issuecomment-475267893
